lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -60,5 +60,10 @@ def ps_script_content(mode)
     template.result(binding)
   end
 
-
+  def ps_invoke_content(mode)
```

Comment:
@jpogran I believe you mentioned that we could just reuse `ps_script_content` given we're fully on board with using `Invoke-DscResource` now... but it looks like `ps_invoke_content` is still kicking around.